hadoop-plugin/src/main/groovy/com/linkedin/gradle/azkaban/AzkabanPlugin.groovy

```diff
@@ -200,6 +201,12 @@ class AzkabanPlugin implements Plugin<Project> {
   }
 
   /**
+   * Prints a message after upload to Azkaban is done
```

Comment:
Can you add a note in the Javadoc comment that this method is intended to be overwritten by subclasses